src/index.ts

```diff
@@ -1,13 +1,22 @@
 import * as express from "express";
 import { Request, Response } from "express";
 
+import userRoutes from "./router/user";
+import authRoutes from "./router/auth";
+import inventoryRoutes from "./router/inventory";
+
 const app = express();
 const { PORT = 3000 } = process.env;
+
+app.use("/auth", authRoutes);
 app.get("/", (req: Request, res: Response) => {
   res.send({
-    message: "hello world ",
+    message: "book store app",
   });
 });
+app.use("/users", userRoutes);
```

Comment:
use `user` instead of users
```suggestion
app.use("/user", userRoutes);
```